### ISSUE TITLE
Can't get stream logger to work

### ISSUE THREAD

**Issue by falmp** [elastic/elasticsearch-js#203]
I'm using winston to log and I can't get elasticsearch logging to work using the stream type. Am I doing something wrong?

**packages.json:**
```javascript
{
  "dependencies": {
    "elasticsearch": "^4.0.1",
    "winston": "^0.9.0",
    "winston-stream": "0.0.0"
  }
}
```

**tests.js:**
```javascript
var elasticsearch = require('elasticsearch');
var winston = require('winston');
var winstonStream = require('winston-stream');
var stream = require('stream');

var logger = new (winston.Logger)({
  transports: [
    new (winston.transports.Console)({
      level: 'debug',
      colorize: true,
      timestamp: true,
      prettyPrint: true
    })
  ]
});

var loggerStream = winstonStream(logger, 'debug');

var client = new elasticsearch.Client({
  apiVersion: '1.5',
  log: {
    level: 'debug',
    type: 'stream',
    stream: loggerStream
  }
});

logger.info('logging initialized sucessfully');
logger.info('is stream instanceof WriteableStream? %s', loggerStream instanceof stream.Writable);

client.info().then(function(result) {
  logger.info('result:', result);
});
```

I suppose this was supposed to log elasticsearch.js requests to console, but I still get the elasticsearch.log file.

**console:**
```
$ node tests.js 
2015-03-29T01:54:42.758Z - info: logging initialized sucessfully
2015-03-29T01:54:42.761Z - info: is stream instanceof WriteableStream? true
2015-03-29T01:54:42.794Z - info: result: 
{ status: 200,
  name: 'M',
  cluster_name: 'elasticsearch',
  version: 
   { number: '1.5.0',
     build_hash: '544816042d40151d3ce4ba4f95399d7860dc2e92',
     build_timestamp: '2015-03-23T14:30:58Z',
     build_snapshot: false,
     lucene_version: '4.10.4' },
  tagline: 'You Know, for Search' }
```

**elasticsearch.log:**
```
INFO: 2015-03-29T01:59:05Z
  Adding connection to http://localhost:9200/

DEBUG: 2015-03-29T01:59:05Z
  starting request { method: 'GET', path: '/', query: {} }
  

DEBUG: 2015-03-29T01:59:05Z
  Request complete
```

**Comment by spalger** [comment]
Well that's embarrassing: https://github.com/elastic/elasticsearch-js/blob/4.0/src/lib/loggers/index.js#L3

**Comment by spalger** [comment]
Just released 4.0.2 with a fix. Can you try it out now?

**Comment by falmp** [comment]
That worked, thank you. :+1: :)
